### ISSUE TITLE
TypeError: Failed to construct 'URL': Invalid URL

### ISSUE THREAD

**Issue by puremana** [dequelabs/axe-core#2825]
This is for the chrome extension "axe - Web Accessibility Testing" which I believe is also developed by dequelabs. I was unsure where else to post this error.

Expectation: Not console error

Actual:
```
Uncaught (in promise) TypeError: Failed to construct 'URL': Invalid URL
    at y (content.bundle.js:33)
```

Motivation: So I can read more important console errors relating to development

<pre><code>
axe - Web Accessibility Testing version: 4.8.4

Browser and Assistive Technology versions
Browser: Chrome 88.0.4324.190 (Official Build) (64-bit)
Operating System: Windows 10

</code></pre>

This occurs on any page, even Google. The error usually occurs 
![image](https://user-images.githubusercontent.com/2869411/109715464-8d788300-7c08-11eb-8f36-31390e72cc45.png)
![image](https://user-images.githubusercontent.com/2869411/109715545-a2551680-7c08-11eb-9d54-65ddc64e3d55.png)

multiple times.

Thanks.

**Comment by LouiseWoodhams24** [comment]
I think I have the solution for this. The URL I think it is having issues with is: https://axe.deque.com which appears in the file a couple if times. If you go to this URL you actually get redirected to https://auth.deque.com/auth/realms/axe/protocol/openid-connect/auth?client_id=axepro-public&redirect_uri=https%3A%2F%2Faxe.deque.com%2F&state=cfeec79f-fbbd-4bbc-8ec3-7ea9b3ea8032&nonce=eab19a6a-0984-4ec3-8143-a4c0f9b43078&response_mode=fragment&response_type=code&scope=openid So I think this URL needs updating

**Comment by stephenmathieson** [comment]
Thanks for reporting!

We've identified the problem and have fixed the issue. The next release of the extension (date TBD) will prevent this from happening.

**Comment by LouiseWoodhams24** [comment]
Was the fix what I thought it was?

**Comment by stephenmathieson** [comment]
Close, but not quite!

We were trying to construct an instance of a `URL` for communicating with the axe API server (https://axe.deque.com) before the server's URL is set. We keep the server's URL in the extension's storage. The code looked something like this:

```js
var stored_url = await get_from_browser_storage('axe_server_url')
var url = new URL(stored_url)
```

The fix resembles:

```js
var stored_url = await get_from_browser_storage('axe_server_url')
var url = new URL(stored_url || DEFAULT_SERVER_URL)
```

**Comment by stephenmathieson** [comment]
Just spoke with @erik-deque. We'll get a release out the door soon(-ish) for you all 😄

**Comment by LouiseWoodhams24** [comment]
Thanks for letting me know the fix :D and super thanks for looking into fixing it so quickly <3

**Comment by puremana** [comment]
Quick fix! Thanks 👍

**Comment by erik-deque** [comment]
Closing as this was fixed in the 4.8.6 release of the extension.
